WordPress/wordpress-playground
# [Data Liberation] Build markdown importer as phar

Builds data-liberation-markdown.phar.gz (200KB) to enable downloading the
Markdown importer only when needed instead of on every page load.

A part of:

* https://github.com/WordPress/wordpress-playground/pull/2080
* https://github.com/WordPress/wordpress-playground/issues/1894

 ## Testing instructions

Run `nx build playground-data-liberation-markdown`, confirm it finished
without errors. A smoke test of the built phar file is included in the
build command.
